config/detekt/detekt.yml

```diff
@@ -0,0 +1,741 @@
+build:
```

Comment:
Is it a default config? Can you please delete it and only keep changes that are different from a default config?